.github/workflows/generate_changelog.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate Changelog
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: 'Release tag of latest release'
+        required: true
+
+permissions:
+      contents: write
+
+jobs:
+  process-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      
+      - name: Install Dependencies
+        run: pip install -r requirements.txt
+          
+      - name: Run Script
+        env:
+          RELEASE_TAG: ${{ inputs.release_tag }}
+          GITHUB_PAT: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        run: python scripts/create_changelog.py "$RELEASE_TAG"
```

Comment:
I think this name `PERSONAL_ACCESS_TOKEN` is too general. How about `SAGE_ACCESS_TOKEN`?

Reply:
Yeah works!